modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml

```diff
@@ -2319,6 +2319,12 @@ components:
                     description:
                         The article's description.
                     type: string
+                displayDate:
```

Comment:
The property name must follow conventions. In this case, I think it should be `datePublished`.
Also, pay attention to the description used in other places; in particular, sentences must end with a period.